scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-4x.yaml

```diff
@@ -1,3 +1,65 @@
+QFN-40-1EP_5x5mm_P0.4mm_EP3.475x3.475mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.issi.com/WW/pdf/31FL3736.pdf#page=28'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    minimum: 4.9
+    nominal: 5
+    maximum: 5.1
+  body_size_y:
+    minimum: 4.9
+    nominal: 5
+    maximum: 5.1
+  overall_height:
+    minimum: 0.7
+    nominal: 0.75
+    maximum: 0.8
+
+  lead_width:
+    minimum: 0.15
+    nominal: 0.2
+    maximum: 0.25
+  lead_len:
+    minimum: 0.3
+    nominal: 0.4
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 3.15
+    maximum: 3.8
+  EP_size_y:
+    minimum: 3.15
+    maximum: 3.8
```

Comment:
The datsheet suggests the use of maximum size for the pad. use the `EP_size_y_overwrite: 3.8` parameter to force the script to use it (same in x direction)

Reply:
Done